charts/incubator/teamspeak3/SCALE/ix_values.yaml

```
@@ -0,0 +1,19 @@
+##
+# This file contains Values.yaml content that gets added to the output of questions.yaml
+# It's ONLY meant for content that the user is NOT expected to change.
+# Example: Everything under "image" is not included in questions.yaml but is included here.
+##
+
+image:
+  repository: teamspeak
+  tag: 3.13.6
```

Please digest pin this:
```suggestion
  tag: 3.13.6@sha256:ff7a92727ffbf05925b54bb12e65bfee311d4193fc68e175dacd63a0048c28bf
```